packages/prism/src/index.js

```diff
@@ -4,12 +4,15 @@
 import Highlight, { defaultProps } from 'prism-react-renderer'
 import { jsx, Styled } from 'theme-ui'
 
-export default ({ children, className: outerClassName, title }) => {
+import Prism from 'prismjs/components/prism-core'
+
+export default ({ children, className: outerClassName, title, prism }) => {
   const [language] = outerClassName.replace(/language-/, '').split(' ')
 
   return (
     <Highlight
       {...defaultProps}
+      Prism={Prism || prism}
```

Comment:
Thanks for looking into this! This *looks like* it would always evaluate to the imported module, but I don't think that's the intent, is it?

Reply:
No @jxnblk ... the intention is to check to see if the user passed the function a ‘prism’ parameter, and if they didn’t, we’ll use the one that we imported

Reply:
Right, but I think @jxnblk's point is that it should be `{prism || Prism}` if I understand this correctly?

Reply:
I think you are right @johno